lib/blueprint_agreement/utils/matcher.rb

```diff
@@ -0,0 +1,48 @@
+require 'minitest/assertions'
+
+module BlueprintAgreement
+  module Utils
+    class Matcher
+      include Minitest::Assertions
```

Comment:
@EdgarOrtegaRamirez  i think this is the main dependency that we have with minitest. i wondering if we can create a new Rspec matcher for `rspec` any thoughts?